client/tests/webdriver/pages/createNewPerson.page.js

```diff
@@ -63,6 +68,54 @@ class CreatePerson extends Page {
     return browser.$(".bp3-datepicker-footer button.bp3-button:first-child")
   }
 
+  get customFieldsContainer() {
+    return browser.$("#custom-fields")
+  }
+
+  get numberCustomFieldContainer() {
+    return this.getCustomFieldContainerByName(INVISIBLE_CUSTOM_FIELDS.number)
+  }
+
+  get numberCustomField() {
+    return this.numberCustomFieldContainer.$('input[type="number"]')
+  }
+
+  get numberCustomFieldHelpText() {
```

Comment:
I think it's good to rename this to numberCustomFieldValidationError. Because of the name, when you did waitForExist in one of the tests I thought the test is not doing anything.

Reply:
It is being used line 228
```
      // Actually see the validation warning
      CreatePerson.numberCustomFieldHelpText.waitForExist()
```